docs/docs/reference/smart_contract_reference/testing/index.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
you also need to include it in sidebars.js

I also think there is a better place to put this. Most places have testing under getting started, but i could see it in reference (not smart contract reference though - feels so nested that no one will find it)

But these are nits that someone from @AztecProtocol/devrel can add in your PR on their own :)

Reply:
You're right, it was hard to find. I moved it next to `Writing Contracts`, but I'll let the devrel team decide it's final home 😄

Reply:
i like where it is next to "writing contracts" in the Guides section